src/events/request/RequestEventHandler.ts

```diff
@@ -24,94 +23,135 @@ export default class RequestEventHandler implements EventHandler<'message'> {
 
 	// This syntax is used to ensure that `this` refers to the `RequestEventHandler` object
 	public onEvent = async ( origin: Message ): Promise<void> => {
-		if ( origin.type !== 'DEFAULT' ) {
-			return;
-		}
-
-		if ( origin.channel instanceof TextChannel ) {
-			this.logger.info( `${ origin.author.tag } posted request ${ origin.id } in #${ origin.channel.name }` );
-		}
-
-		try {
-			await origin.reactions.removeAll();
-		} catch ( error ) {
-			this.logger.error( error );
-		}
-
-		const regex = new RegExp( `(?:${ MentionCommand.getTicketLinkRegex().source }|(${ MentionCommand.ticketPattern }))(\\?\\S+)?`, 'g' );
+		if ( !BotConfig.request.testingRequestChannels || !BotConfig.request.testingRequestChannels.includes( origin.channel.id ) ) {
```

Comment:
I don't like this large if statement that wraps around essentially the whole function.